### ISSUE TITLE
Add settings toggle for tanoy annoucements

### ISSUE THREAD

**Issue by unspec** [kalleth/mpukviewr#12]
Tannoy messages (and service announcements) is the only thing you cannot toggle

**Comment by kalleth** [comment]
Service announcements is deliberate. You will not ignore me, dammit! I'll speak to MPUK about tannoy, though :)

**Comment by unspec** [comment]
Yeah I figured service announcements should stay compulsory so i left it out of the title.

As you can toggle all of the actual 'content' source options it seemed like tannoy should be in there for consistency. I cant actually imagine anybody wanting to turn it off though as its likely to have a good signal-to-noise ratio.
